uast/uast.go

```diff
@@ -33,6 +33,21 @@ const (
 	// File is the root node of a single file AST.
 	File
 
+	// Binary aritmethic operators. Examples with C operators.
+	// TODO: should we have division and FloorDivision like Python or Nim?
+	// TODO: should we had the pow operator that some languages have?
+
+	// OpArithmeticAdd is the binary add operator (i.e. + in most languages)
+	OpArithmeticAdd
+	// OpArithmeticSubstract is the binary subtract operator (i.e. - in most languages)
+	OpArithmeticSubstract
+	// OpArithmeticMultiply is the binary multiply operator (i.e. * in most languages)
+	OpArithmeticMultiply
+	// OpArithmeticDivision is the binary division operator (i.e. / in most languages)
+	OpArithmeticDivision
```

Comment:
There are two ways to call these things:

|  Action       |  Name          |
| ------------- |-------------|
| Add      | Addition |
| Subtract | Subtraction |
| Multiply | Multiplication |
| Divide | Division |

Right now we are mixing both, we should choose one.